spec/non_conformant/parser/operations/addition/dimensions/pairs.hrx

```diff
@@ -266,3 +266,403 @@ foo {
+
+but you may have intended it to mean:
+
+    10px (+10px)
+
+Add a space after + to clarify that it's meant to be a binary operation, or wrap
+it in parentheses to make it a unary operation. This will be an error in future
+versions of Sass.
+
+More info and automated migrator: https://sass-lang.com/d/strict-unary
+   ,
+27 |   test-26: 10px +10px;
+   |            ^^^^^^^^^^
+   '
+
+DEPRECATION WARNING on line 31, column 12 of input.scss: 
+This operation is parsed as:
+
+    10px + #{10}px
+
+but you may have intended it to mean:
+
+    10px (+#{10}px)
+
+Add a space after + to clarify that it's meant to be a binary operation, or wrap
+it in parentheses to make it a unary operation. This will be an error in future
+versions of Sass.
+
+More info and automated migrator: https://sass-lang.com/d/strict-unary
+   ,
+31 |   test-30: 10px +#{10}px;
+   |            ^^^^^^^^^^^^^
+   '
+
+DEPRECATION WARNING on line 35, column 12 of input.scss: 
+This operation is parsed as:
+
+    10px + 1
+
+but you may have intended it to mean:
+
+    10px (+1)
+
+Add a space after + to clarify that it's meant to be a binary operation, or wrap
+it in parentheses to make it a unary operation. This will be an error in future
+versions of Sass.
+
+More info and automated migrator: https://sass-lang.com/d/strict-unary
+   ,
+35 |   test-34: 10px +1#{0}px;
+   |            ^^^^^^^
+   '
+
+DEPRECATION WARNING on line 39, column 12 of input.scss: 
+This operation is parsed as:
+
+    10px + 10
+
+but you may have intended it to mean:
+
+    10px (+10)
+
+Add a space after + to clarify that it's meant to be a binary operation, or wrap
+it in parentheses to make it a unary operation. This will be an error in future
+versions of Sass.
+
+More info and automated migrator: https://sass-lang.com/d/strict-unary
+   ,
+39 |   test-38: 10px +10#{px};
+   |            ^^^^^^^^
+   '
+
+DEPRECATION WARNING on line 43, column 12 of input.scss: 
+This operation is parsed as:
+
+    10px + 10
+
+but you may have intended it to mean:
+
+    10px (+10)
+
+Add a space after + to clarify that it's meant to be a binary operation, or wrap
+it in parentheses to make it a unary operation. This will be an error in future
+versions of Sass.
+
+More info and automated migrator: https://sass-lang.com/d/strict-unary
+   ,
+43 |   test-42: 10px +10#{p}x;
+   |            ^^^^^^^^
+   '
+
+DEPRECATION WARNING on line 47, column 12 of input.scss: 
+This operation is parsed as:
+
+    #{10}px + #{10}px
+
+but you may have intended it to mean:
+
+    #{10}px (+#{10}px)
+
+Add a space after + to clarify that it's meant to be a binary operation, or wrap
+it in parentheses to make it a unary operation. This will be an error in future
+versions of Sass.
+
+More info and automated migrator: https://sass-lang.com/d/strict-unary
+   ,
+47 |   test-46: #{10}px +#{10}px;
+   |            ^^^^^^^^^^^^^^^^
+   '
+
+DEPRECATION WARNING on line 51, column 12 of input.scss: 
+This operation is parsed as:
+
```

Comment:
It looks like this going over 500 lines is causing the linter to fail

Reply:
Should be fixed by https://github.com/sass/sass-spec/pull/1824